scripts/update-integration-test-coordinates.sh

```diff
@@ -0,0 +1,2 @@
+#!/bin/bash
+python integration-test.py -printcoords | python scripts/update-integration-test-coordinates.py
```

Comment:
Btw, I don't mean to suggest that the way it is at the moment is wrong. When I read it, I thought "is `integration-test.py` in the `scripts/` directory?" and it took me a moment to figure out what was going on. I think it's better to avoid confusion (perhaps it's only my confusion :confounded: ) by being explicit about the locations of files where we can.